swap/src/asb/event_loop.rs

```diff
@@ -176,13 +176,13 @@ where
                             let _ = self.handle_execution_setup_done(peer_id, swap_id, *state3).await;
                         }
                         SwarmEvent::Behaviour(OutEvent::SwapDeclined { peer, error }) => {
-                            tracing::warn!(%peer, "Ignoring spot price request because: {}", error);
+                            tracing::warn!(%peer, "Ignoring spot price request: {}", error);
```

Comment:
```suggestion
                            tracing::warn!(%peer, "Ignoring spot price request: {:#}", error);
```

Reply:
It is not an `anyhow::Error` so I don't think the alternate `Display` is going to do anything useful.